chapters/synchronous.tex

```diff
@@ -565,32 +595,42 @@ \subsection{Base-clock conversion operators}\label{base-clock-conversion-operato
 hold($u$)
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
-Input argument $u$ is a clocked Component Expression (see \cref{argument-restrictions-component-expression}) or a parameter expression.  The operator returns a piecewise constant signal of the same type of $u$.  When the clock of $u$ ticks, the operator returns $u$ and otherwise returns the value of $u$ from the last clock activation.  Before the first clock activation of $u$, the operator returns the start value of $u$, see \cref{initialization-of-clocked-partitions}.
+Input argument $u$ is a clocked component expression (\cref{def:component-expression}) or a parameter expression.
+The operator returns a piecewise constant signal of the same type of $u$.
```

Comment:
This was wrong before as well, but it should be "of the same type _as_ $u$", right?

Reply:
Done.  Resolving.